docs/presetGuides.md

```diff
@@ -297,3 +253,129 @@ nodes:
     - brokerDockerComposeDebugMode: false # excluding the broker
 ````
 
+
+## Private Keys and Security 
+
+The following sections describes the different ways you can encrypt and protect your nodes's keys. 
+
+### Specify the Nodes' Private Keys.
+
+If you know the Private Keys of your node, you can provide them in a custom preset:
+
+#### **`plain-custom-preset.yml`**
+```yaml
+nodes:
+    - voting: true
+      mainPrivateKey: CA82E7ADAF7AB729A5462A1BD5AA78632390634904A64EB1BB22295E2E1A1BDD
+      transportPrivateKey: 6154154096354BC3DB522174ACD8BFE553893A0991BD5D105599846F17A3383B
+      remotePrivateKey: E27AD508907524E2143EF2A3A272DDBEE7558B92550ABA5B82AD65D66B57BD00
+      vrfPrivateKey: F3C24C153783B683E40FB2671493B54480370BF4E3AB8027D4BF1293E14EB9B8
+      votingPrivateKey: EFE3F0EF0AB368B8D7AC194D52A8CCFA2D5050B80B9C76E4D2F4D4BF2CD461C1
+```
+
+Usage examples:
+
+-   Rebuild one of your Testnet/Main nodes.
+-   Create a new node from migrated opted in accounts.
+
+If your custom preset contains private keys, it's highly recommended to `encrypt` it and provide `--password` when starting or configuring the node with Bootstrap.
+
+```
+symbol-bootstrap encrypt --password 1234 --source plain-custom-preset.yml --destination encrypted-custom-preset.yml
+rm plain-custom-preset.yml
+symbol-bootstrap start --password 1234 --preset testnet --assembly dual --customPreset encrypted-custom-preset.yml
+```
+
+Remember to always keep your custom preset, the target's `addresses.yml`, and `preset.yml` privately!
+
+### Specify the Nodes' Public Keys.
+
+In addition to the above, you can provide the accounts Public Keys instead of Private keys in your custom preset.
+This adds a level of security as the private keys aren't in the custom preset. 
+Bootstrap will prompt for them when performing the different operations requiring user intervention when running the commands.
```

Comment:
```suggestion
In addition to the above, you can provide the accounts public keys instead of the private keys in your custom preset.
This adds a level of security as the private keys aren't present in the custom preset. 
Bootstrap will request them when performing the different operations requiring user intervention when running the commands.
```